tox.ini

```diff
@@ -1,8 +1,7 @@
 [tox]
 envlist =
-    flake8
-    isort
-    py{39.310,311}-dj{32,40,41}-sqlite
+    ruff
```

Comment:
Where is the ruff config? I may be missing something, but can't see it.

Reply:
@marksweb The ruff config resides in `pyproject.toml`. I just added the ruff dependency to the testing requirements for easier testing. Transition to ruff was in #329